internal/ethapi/api.go

```diff
@@ -903,7 +903,7 @@ func DoEstimateGas(ctx context.Context, b Backend, args CallArgs, blockNrOrHash
 		return 0, err
 	}
 	// For now always return the gas limit
-	return hexutil.Uint64(block.GasLimit() - 1), nil
+	return hexutil.Uint64(block.GasLimit() - 2000000), nil
```

Comment:
Note: the `2,000,000` gas is analogous to accounting for the intrinsic gas used during the tx